README.md

```diff
@@ -18,7 +18,7 @@ The [TAG Contributor Strategy charter](https://github.com/cncf/toc/blob/main/tag
 We have one general meeting a month, along with meetings for our working groups.
 Go to [the CNCF calendar](https://tockify.com/cncf.public.events/monthly?search=Contributor+Strategy) to see when we meet next.
 
-- [Meeting minutes and agenda](https://bit.ly/cncf-contribstrat-agenda)
+- [Meeting minutes and agenda](https://bit.ly/meetingagendacncf)
```

Comment:
I think we want to use a google docs link and not a bit.ly link we can't update.  That's one of the problems with the old link as well.  Can you just put the gdocs link here?